lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -29,41 +29,73 @@ module AuthenticatableExtensions
         extend ActiveSupport::Concern
 
         included do
+          multi_email_association.configure_autosave!{ include AuthenticatableAutosaveExtensions }
```

Comment:
That's why I will always put brackets around functions. Unless the function is supposed to be a DSL.

I generally follow the style guide of airbnb: https://github.com/airbnb/ruby#